src/context/WidgetSettingContext.tsx

```diff
@@ -97,49 +98,83 @@ export const WidgetSettingProvider = ({
       return false;
     })(cachedSession);
 
-    const response = await getWidgetSetting({
+    const sessionKey = await (async () => {
+      // TODO: new manual strategy
+      return undefined;
+      // if (!injectedUserId || !configureSession) return undefined;
+      // if (strategy !== 'manual' || useCachedSession) return undefined;
+      // try {
+      //   const handler = configureSession({} as any);
+      //   if (handler.onSessionTokenRequired) {
+      //     return (
+      //       (await new Promise(handler.onSessionTokenRequired)) ?? undefined
+      //     );
+      //   }
+      // } catch {
+      //   // noop
+      // }
+      // return undefined;
+    })();
```

Comment:
so for manual, bot styles will always be fetched from now on?

Reply:
Before and after, the `botStyle` always be fetched.

Reply:
ah before was also returning undefined. the old logic is hard to grasp. Might as well just remove it and change to
```
 await widgetSettingHandler(strategy, useCachedSession, {
      host: apiHost,
      appId,
      botId,
      userId: strategy === 'manual' ? injectedUserId : cachedSession?.userId,
      sessionKey: undefined,
    })
```